rsksmart/rskj
# Make rocksdb the default DB instead of leveldb

## Description
* Added rocksdb as default in the .conf files.
* Updated the default return type of `getDbKindValueFromDbKindFile`

## Motivation and Context
Currently, RSKj default library storage is LevelDb. 

We want to make RocksDb the default db. 

Nodes running on LevelDb should keep using LevelDb, same with RocksDb. But nodes started from scratch will use RocksDb.

## How Has This Been Tested?

1. Running node from scratch and then stop it.
2. Run the node again.
3. Run node from scratch selecting non-default DB and then stop it.
4. Run the node again.
5. Run the node again with a different db selected and make sure it was running fine as well.

## Types of changes


## Checklist:


- [ ] Requires Activation Code (Hard Fork)


* **Other information**:
